lib/procore/requestable.rb

```diff
@@ -363,8 +361,15 @@ def multipart?(body)
       RestClient::Payload::has_file?(body)
     end
 
-    def full_path(path)
-      File.join(base_api_path, path).to_s
+    def full_path(path, version)
+      version ||= options[:default_version]
+      if version == "vapid"
+        File.join(base_api_path, "/vapid", path)
+      elsif /^v\d+\.\d+$/.match?(version)
+        File.join(base_api_path, "/rest/#{version}", path)
```

Comment:
Change to `File.join(base_api_path, "rest", version, path)` to remove the un-needed extra interpolation